lib/common_test/src/ct_logs.erl

```diff
@@ -1637,6 +1674,7 @@ index_header(Label, StartTime, CustomStylesheet) ->
       "<th>Failed</th>\n",
       "<th>Skipped", xhtml("<br>", "<br />"), "(User/Auto)</th>\n"
       "<th>Missing", xhtml("<br>", "<br />"), "Suites</th>\n",
+	  "<th>Suite", xhtml("<br>", "<br />"), "Time</th>\n",
```

Comment:
I think `Suite Time` is not optimal.
AFAIK, if a test spec is executed it will be represented as a single row as well, but might include multiple suites.

Maybe name should be more general to cover also such case ? `Elapsed Time` ?

Reply:
Changed to Elapsed Time